fe/fe-core/src/main/java/org/apache/doris/tablefunction/ExternalFileTableValuedFunction.java

```diff
@@ -180,6 +193,74 @@ protected void parseProperties(Map<String, String> validParams) throws AnalysisE
         stripOuterArray = Boolean.valueOf(validParams.get(STRIP_OUTER_ARRAY)).booleanValue();
         numAsString = Boolean.valueOf(validParams.get(NUM_AS_STRING)).booleanValue();
         fuzzyParse = Boolean.valueOf(validParams.get(FUZZY_PARSE)).booleanValue();
+
+        if (formatString.equals("csv") || formatString.equals("csv_with_names")
+                || formatString.equals("csv_with_names_and_types")) {
```

Comment:
For `csv_with_names_and_types` type, may be we should parse csv schema from csv file?

Reply:
Yes, may be later, because I didn't find a standard format of csv names and types yet.
And I think if user specified the schema, it should be override the names and type in file